lib/tasks/taxonomy/find_nested_tags.rake

```diff
@@ -0,0 +1,124 @@
+require 'set'
+require 'csv'
+
+namespace :taxonomy do
+  desc "Find any content tagged at one taxon level, then tagged again at one of those taxon's ancestors"
+  task :find_nested_tags, [:root_taxon_id] => [:environment] do |_, args|
+    root_taxon_id = args.fetch(:root_taxon_id)
+    root_taxon = OpenStruct.new Services.publishing_api.get_content(root_taxon_id).to_h
+    taxonomy = Taxonomy::ExpandedTaxonomy.new(root_taxon.content_id)
+    taxonomy.build
+
+    nested_tag_warnings = find_content_tagged_at_multiple_levels(taxonomy.child_expansion)
+
+    output = format_warnings_as_rows nested_tag_warnings
```

Comment:
Could you please wrap arguments in parenthesis? It' makes it easier to read and consistent with other method calls.